mythril/interfaces/cli.py

```diff
@@ -364,10 +365,16 @@ def main():
             print(storage)
 
         elif args.disassemble:
-            easm_text = mythril.contracts[
-                0
-            ].get_easm()  # or mythril.disassemble(mythril.contracts[0])
-            sys.stdout.write(easm_text)
+            output = ""
+            easm_text = mythril.contracts[0].get_easm()
+            # or mythril.disassemble(mythril.contracts[0])
+
+            if easm_text:
+                output += "Runtime Disassembly: \n" + easm_text
+            if code:
+                output += "Disassembly: \n" + mythril.contracts[0].get_creation_easm()
+
+            sys.stdout.write(output)
```

Comment:
Why not print?

Reply:
I think you want keep `sys.stdout.write` here. The `print()` adds an extra newline.